src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -34,12 +64,36 @@ public boolean isTestOnly() {
         return testOnly;
     }
 
+    @Override
+    public String getCaseNumber() {
+        return caseNumber;
+    }
+
+    @Override
+    public Classification getClassification() {
+        return classification;
+    }
+
+    @Override
+    public String getJurisdiction() {
+        return jurisdiction;
+    }
+
+    @Override
+    public List<String> getDocumentUrls() {
+        return documentUrls;
+    }
+
     @Override
     public String toString() {
         return "EnvelopeMsg{"
-            + "envelopeId='" + envelopeId + '\''
-            + "testOnly='" + testOnly + '\''
-            + '}';
+            + "envelopeId='" + envelopeId + "'"
```

Comment:
maybe we could use this so we don't have to maintain toString fields?
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ReflectionToStringBuilder.html

Example of using it:
https://github.com/hmcts/cmc-claim-store/blob/40234e54341097e7c21473cb23138038bbe5032e/domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Timeline.java#L33
https://github.com/hmcts/cmc-claim-store/blob/812f7aef900c09be93d3ebc57b3fd21dc471e0a8/domain-model/src/main/java/uk/gov/hmcts/cmc/domain/utils/ToStringStyle.java#L10

Reply:
Let's not log all the details here for e.g caseNumber. For now we could just log zip filename and envelope id.

Reply:
@nitinprabhu Sure, I'll add zip filename and the log message. Is the "zip file name" used just for logging or in the queue Message as well?

Reply:
yes it is needed in queue message and also need to make sure it is in orchestrator as well :)